guides/it/Miro.md

```diff
@@ -0,0 +1,43 @@
+# Miro
+
+All the information below and lots more useful stuff can be found on the Made Tech [Miro training board](https://miro.com/app/board/o9J_lkgJ4BU=/) (this is an internal link to a Miro board)
+
+## Getting access to Miro
+### New starters
+All new starters after November 2021 are automatically given a Free Restricted Miro licence when they join and are added to the Made Tech Team. This gives them the ability to view and comment with the option to request editing access.
+
+They can be added to any board or project in the Made Team Team by any other Made Tech Team member. To add them to another project or client Team, that Team Admin has to invite them.
+
+If they need access to edit then they can request it via Miro and a Miro Admin will be notified and grant that asap.
+
+### Been here a while
```

Comment:
would suggest making this a bit clearer; i.e. more explicitly saying 'if you joined before November 2021 and don't currently have a license'

Reply:
Updated